src/main/resources/mappers/CityMapper.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper
+  PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+  "https://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="main.java.com.solvd.laba.db.mapper.CityMapper">
+	
+  <select id="selectCityById" resultMap="main.java.com.solvd.laba.db.mapper.CityMapper.cityResultMap">
+    select *
+	from cities C
+	join states S on C.state_id = S.state_id
+	where C.city_id = #{id}
+  </select>
+  
+   <select id="selectCityByName" resultMap="main.java.com.solvd.laba.db.mapper.CityMapper.cityResultMap">
+    select *
+	from cities C
+	join states S on C.state_id = S.state_id
+	where C.city_name = #{name}
+  </select>
+  
+   <select id="selectCities" resultMap="main.java.com.solvd.laba.db.mapper.CityMapper.cityResultMap">

```

Comment:
for the `resultMap`, you can shorten the name of it because if you have defined a result map within your mapper XML file, you can refer to it directly by its id instead of specifying the full package and class name.